cmd/sysl/cmd_env.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+const KnownEnv = `
+	SYSL_MODULES
+	SYSL_PLANTUML
+`
+
+type envCmd struct{}
+
+func (c *envCmd) Name() string       { return "env" }
+func (c *envCmd) MaxSyslModule() int { return 0 }
+
+func (c *envCmd) Configure(app *kingpin.Application) *kingpin.CmdClause {
+	cmd := app.Command(c.Name(), "Print sysl environment information.")
+	return cmd
```

Comment:
`return app.Command(c.Name(), "Print sysl environment information.")`

Reply:
Yes. If you run `sysl env --help`, the output will show the explanation.